scripts/automated-patch-releases.md

```diff
@@ -0,0 +1,49 @@
+# CRI-O Automated Patch Releases
+
+Automated Patch Releases provides an easy way of packaging and release new version
+of CRI-O
+This involves the use of Golang Scripts and Github Actions
+
+```mermaid
+    flowchart TD
+      A[Start: patch-release.yml Workflow] --> B[Create version bump intent prs]
+      B --> C[Maintainers Review PRs]
+      C -->|Decides Against Merges| E[Rebases on main branch] --> C
+      C -->|Decides to Merge PRs| D[Run: tag-reconciler.yml Workflow]
+      D --> F[Release Job: Build Packages, Create Release Notes, etc.]
+      F --> G[End]
```

Comment:
That's not the end. This reconcile job runs every day to create and publish the packages: https://github.com/cri-o/packaging/actions/workflows/schedule.yml

Reply:
What triggers the reconcile job, is it the <a href="https://github.com/cri-o/cri-o/actions/workflows/test.yml">test.yml</a> or is there something else that emits an event for that to run?

Reply:
A cronjob: https://github.com/cri-o/packaging/blob/2b2aeaeaad98f87239e34f850e73b5957a41a111/.github/workflows/schedule.yml#L5

Reply:
okay but I am struggling to see how the workflows here, patch-release or  the test.yml which is last bit to run, relates to this schedule.yml. It looks like they are independent of each other. 
Also the schedule.yml is in a different repo so It's hard for me to visualize the relationship.